source/api_reference/index.html.md.erb

```diff
@@ -3,54 +3,58 @@ title: API reference
-
-The same base URL is now used for testing and live. The API key you use determines if the actions are treated as sandbox test payments or processed as real payments.
+The API is based on REST principles. It returns data in JSON format, and uses
+standard HTTP error response codes. 
 
-For full details of each API action, see the API Browser:
+For full details of each API action, see the API browser: 
 
-- <a href="https://gds-payments.gelato.io/docs/versions/1.0.2/resources/general" target="blank">General functions</a> (link opens in new window)
-- <a href="https://gds-payments.gelato.io/docs/versions/1.0.2/resources/payment-id" target="blank">Payment ID functions</a> (link opens in new window)
+- <a href="https://gds-payments.gelato.io/docs/versions/1.0.2/resources/general" target="blank">General functions</a> [external link] 
+- <a href="https://gds-payments.gelato.io/docs/versions/1.0.2/resources/payment-id" target="blank">Payment ID functions</a> [external link] 
 
-You can also use our interactive <a href="https://gds-payments.gelato.io/api-explorer/gov-uk-pay-api/versions/1.0.2//" target="blank">API Explorer</a> (link opens in new window) to try out API calls and view responses.
+You can also use the interactive <a
+href="https://gds-payments.gelato.io/api-explorer/gov-uk-pay-api/versions/1.0.2//"
+target="blank">API explorer</a> [external link] to try out API calls and view
+responses.
 
-See the [Quick Start Guide](/quick_start_guide/#quick-start-guide) section for how to set up the API Explorer. Make sure you enter your sandbox API key to avoid generating real payments.
+See the ["Quick start guide"](/quick_start_guide/#quick-start-guide) section for
+how to set up the API explorer. Make sure you enter your sandbox API key to
+avoid generating real payments.
 
+## Authentication
 
+GOV.UK Pay authenticates API calls with OAuth2 HTTP bearer tokens. These are
+easy to use and contain only your API key. [RFC
+6750](http://tools.ietf.org/html/rfc6750) details bearer tokens.  
 
-## API authentication
-
-GOV.UK Pay authenticates API calls with OAuth2 HTTP bearer tokens. These are easy to use and consist of one component: your API key. Bearer tokens are specified in [RFC 6750](http://tools.ietf.org/html/rfc6750).
-
-When making an API call, you’ll need to add your API key to an "Authorization" HTTP header and prefix it with "Bearer ". This is an example of how a header would look.
+When making an API call, you need to add your API key to an "Authorization"
+HTTP header prefixed with "Bearer ". For example: 
 
 ```text
-Authorization: Bearer 2o9bkusxaicha5c9gnsj790of7asggo1feraoufbai4a
+Authorization: Bearer <YOUR-API-KEY-HERE> 
 ```
 
-
 ## Payment status lifecycle
 
-This diagram gives an overview of the payment status lifecycle and the possible outcomes.
+This diagram gives an overview of the payment status lifecycle and the
+possible outcomes.
 
 ![](https://s3-eu-west-1.amazonaws.com/pay-govuk-documentation/pay-transaction-states.png)
```

Comment:
Is this still a valid diagram?

Reply:
yes it is valid (apart from transactions with delayed capture), although personally i find it quite confusing and would love to get it changed at some point! it doesn't clearly show what initiates the state changes

Reply:
it is valid (though not for transactions with delayed capture). but would be great to change it sometime because it's not clear about what action initiates a state change...

Reply:
OK, so we could use this for the time being, and say that it's not valid for delayed capture?

Reply:
yes